### ISSUE TITLE
Add button to test CI/CD pipeline

### ISSUE THREAD

**Issue by lo-b** [broscience-lab/peaceful-ares-webapp#10]
Add a simple button to webapp, create a PR, merge and check if CI/CD
is working as expected, in particular:

- Run CI on push to dev branches
- Run delivery only after merge
- Run deploy after delivery, trigger IaC edit in GCP repo -- which in turn
  should run CI/CD there, creating a new plan and applying it.
